Looms.PoS/Looms.PoS.Application/Features/Reservation/Commands/UpdateReservation/UpdateReservationRequestValidator.cs

```diff
@@ -0,0 +1,58 @@
+using FluentValidation;
+using Looms.PoS.Application.Models.Requests.Reservation;
+using Looms.PoS.Application.Utilities.Validators;
+using Looms.PoS.Application.Utilities;
+using Looms.PoS.Domain.Interfaces;
+using Looms.PoS.Domain.Exceptions;
+
+namespace Looms.PoS.Application.Features.Reservation.Commands.UpdateReservation;
+
+public class UpdateReservationRequestValidator : AbstractValidator<UpdateReservationRequest>
+{
+     private readonly IServicesRepository _servicesRepository;
```

Comment:
not needed property

Reply:
you mean servicesRepository?

Reply:
yes

Reply:
not needed because it is not used

Reply:
but in service id validation we are using it?

Reply:
use the argument that comes in constructor

Reply:
deleted the property and now the arg is used